docs/tutorials/emulation/01_quickstart.ipynb

```diff
@@ -105,7 +105,7 @@
    "outputs": [],
    "source": [
     "# Run AutoEmulate with default settings\n",
-    "ae = AutoEmulate(x, y, log_level=\"error\")"
+    "ae = AutoEmulate(x, y, log_level=\"error\", n_iter=3)"
```

Comment:
I feel it would be better if the Quickstart wasn't using any custom settings (setting the log level is fine), the point here is to show that one can use `AutoEmulate` as is without having to worry about setting anything.

Reply:
Okay fair enough